core/storage/base_model/gae_models.py

```diff
@@ -267,6 +282,23 @@ def get_all_commits(cls, page_size, urlsafe_start_cursor):
         return cls._fetch_page_sorted_by_last_updated(
             cls.query(), page_size, urlsafe_start_cursor)
 
+    @classmethod
+    def get_commit(cls, instance_id, version):
+        """Returns the commit corresponding to an instance id and
+        version number.
+
+        Args:
+            instance_id: str. The id of the particular construct, from which
```

Comment:
The effect of naming both the arg and the function above as instance_id has the consequence that now, I don't know what instance_id actually refers to :)

Probably it means the same thing as above, in which case please see previous comment.

Reply:
Yeah, it was. Renamed it.